Directory.Packages.props

```diff
@@ -10,11 +10,15 @@
   <ItemGroup>
     <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorVersion)" />
     <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsVersion)" />
-    <PackageVersion Include="System.Memory" Version="4.5.5" />
+
+    <!-- These versions are not updated by dependency flow as they aren't produced live anymore. -->
+    <PackageVersion Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6" />
```

Comment:
For a follow-up PR: We should remove this test only dependency as it brings in a very old version of RuntimeInformation on .NET Framework which will eventually get marked as deprecated.

I don't think we need it. dotnet/runtime doesn't depend on it anymore and also use the TestUtilities.cs files.